components/org.wso2.micro.integrator.initializer/src/main/java/org/wso2/micro/integrator/initializer/ServiceBusInitializer.java

```diff
@@ -96,6 +102,10 @@
 
     private TaskService taskService;
 
+    private static DataSourceService dataSourceService;
+
+    private TransactionHandlerComponent transactionHandlerComponent;
```

Comment:
What abt `TansactionCountHandler`? IMO, `TransactionHandler` itself gives a broader idea. We are not handling transactions. We just count here. Isn't it?

Reply:
+1

Reply:
+1